README.md

```diff
@@ -118,26 +118,250 @@ services:
+
+```
+$ pebble run --verbose
+2022-10-26T01:41:32.805Z [pebble] Started daemon.
+2022-10-26T01:41:32.835Z [pebble] POST /v1/services 29.743632ms 202
+2022-10-26T01:41:32.835Z [pebble] Started default services with change 7.
+2022-10-26T01:41:32.849Z [pebble] Service "srv1" starting: python3 -u /path/to/srv1.py
+2022-10-26T01:41:32.866Z [srv1] This is log number 0
+2022-10-26T01:41:35.870Z [srv1] This is log number 1
+2022-10-26T01:41:38.873Z [srv1] This is log number 2
+...
+```
+
+### Starting and stopping services
+
+To start or stop specific services, type `pebble start` or `pebble stop` followed by one or more service names, for example:
+
+```
+$ pebble start srv1 srv2  # start two services (and any dependencies)
+
+$ pebble stop srv1        # stop one service
+```
+
+When starting a service, Pebble records a "Start" [change](#changes-and-tasks) for the operation, executes the service's `command`, and waits 1 second to ensure the command doesn't exit too quickly. Assuming the command doesn't exit within 1 second, the start is considered successful, otherwise `pebble start` will exit with an error.
+
+When stopping a service, Pebble records a "Stop" change for the operation, sends SIGTERM to the service's process group, and waits up to 5 seconds. If the command hasn't exited after 5 seconds, Pebble sends SIGKILL to the service's process group and waits up to 5 more seconds. If the command exits within that 10 seconds, the stop is considered successful, otherwise `pebble stop` will exit with an error.
+
+You can view the status of one or more services by using `pebble services`:
+
+```
+$ pebble services srv1       # show status of a single service
+Service  Startup  Current
+srv1     enabled  active
+
+$ pebble services            # show status of all services
+Service  Startup   Current
+srv1     enabled   active
+srv2     disabled  inactive
+```
+
+The "Startup" column shows whether this service should be automatically started when Pebble starts ("enabled" means auto-start, "disabled" means don't auto-start).
+
+The "Current" column shows the current status of the service, and can be one of the following:
+
+* `active`: starting or running
+* `inactive`: not yet started, being stopped, or stopped
+* `backoff`: in a [backoff-restart loop](#service-auto-restart)
+* `error`: in an error state (currently: exited and exit action is "ignore")
+
+Implementation note: Pebble's service manager uses a [state machine](https://github.com/canonical/pebble/blob/a1f52239079e58cb6e3b6c7472ef5438b91eefab/internal/overlord/servstate/handlers.go#L67-L82) to manage the life cycle of a service. You can see these internal states and their transitions in the [state diagram](https://raw.githubusercontent.com/canonical/pebble/master/internal/overlord/servstate/state-diagram.svg).
```

Comment:
Similar to the above point, we don't want to be documenting internal details of the implementation and pointing people to the code in a section that is intended for a new user that has never seen Pebble.

Reply:
Fair -- I've removed this paragraph.